package.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
This project has a package-lock.json which means it was created using npm. All other repos use yarn. We should stick to using yarn